src/sentry/integrations/msteams/webhook.py

```diff
@@ -52,10 +52,57 @@ def post(self, request):
                     absolute_uri("/extensions/msteams/configure/"),
                     signed_params,
                 )
-                # TODO: Better message
+                # TODO: Refactor message creation
+                # TODO: Tweak welcome message appearance to perfection
+                logo = {
+                    "type": "Image",
+                    "url": "https://sentry-brand.storage.googleapis.com/sentry-glyph-black.png",
+                    "size": "Medium",
+                }
+                welcome = {
+                    "type": "TextBlock",
+                    "weight": "Bolder",
+                    "size": "Large",
+                    "text": "Welcome to Sentry for Teams!",
+                    "wrap": True,
+                }
+                description = {
```

Comment:
I think we can put these constant objects into a separate file and import when needed.

Reply:
That's fair, but @scefali just suggested I get it done and refactor later.

Reply:
@KKaempen I think I didn't explain this well. There should be a separate function for sending the message. Like `send_wecome_message`. We just don't need a function to handle sending generic messages.

Reply:
Gotcha, OK. Give me a sec.

Reply:
Changed things up, sorry about misunderstanding. Let me know if there's something else I need to change.